### DISCUSSION TITLE
XRMToolbox New Plugin review

### DISCUSSION THREAD

**Discussion by YesWeCandrew** [MscrmTools/XrmToolBox#1351] [General]
Hi all,

Firstly a big thanks to this great community - XRMToolbox is such a useful app, thanks for all the time put into develop and maintain it. I've wanted to become a part of that and have been been updating the plugin [DataImport++](https://www.xrmtoolbox.com/plugins/JosephMerhej.DataImport/). 

I've made some updates and improvements and have met with Joseph - we're ready to release a BETA updated version of the app, and I've submitted it to the [portal](https://www.xrmtoolbox.com/SignIn?ReturnUrl=%2Fplugins%2Fnew%2F) a few weeks ago but it's still in review. Does anyone know roughly how long it takes for new plugins to get reviewed? I know everyone's a volunteer so I don't mean to rush things I was just hoping I could get a bit of an estimate 😄 

You can check out the [BETA version here](https://www.xrmtoolbox.com/plugins/AndrewHowes.DataImportSharp/).

Thank you!

**Comment by daryllabar** [comment]
I think you happened to hit the late summer holiday for Tanguy...  I had a PR that I am waiting on him returning to approve.

**Comment by crsnt** [comment]
I also have a plugin pending for validation.
Also I think the Open source flag had a bug.. I provided Github project URL but it was flagged as false.

**Reply by MscrmTools** [reply]
For this concern, it can be a mistake of mine as it's a manual process. But providing just a Github Url when the repository is private does not make a tool an open source one. I don't know if it's your case or not. If not please tell me the name of your plugin so that I can fix that
